docker/docker-run.sh

```diff
@@ -0,0 +1,20 @@
+#!/bin/bash
+
+CMD="docker run"
+
+if [ "$1" = "-d" ]; then
+  CMD="$CMD -d --restart always"
+else
+  CMD="$CMD -ti --rm"
+fi
+
+CMD="$CMD --name mojopaste"
+CMD="$CMD -v $PWD:/usr/src/app"
+CMD="$CMD -p 5555:8080"
+CMD="$CMD mojopaste_v1"
+# NOTE: Start by deamon
+# CMD="$CMD sh -c 'cd /usr/src/app && exec mojopaste daemon --listen http://*:8080'"
+# NOTE start by hypnotoad
+CMD="$CMD sh -c 'cd /usr/src/app && exec hypnotoad -f /usr/local/bin/mojopaste'"
+
+eval $CMD
```

Comment:
Is this generic enough to have in this project?

Reply:
We can build docker image and pull it to hub.docker.com
or build it on localhost and after that use docker run 
Expamle https://hub.docker.com/_/wordpress/
`docker run --name some-wordpress --link some-mysql:mysql -d wordpress`

Or we can use docker-compose. But I think its overhead